lib/utils/errors.js

```diff
@@ -37,3 +37,9 @@ module.exports.USERNAME_EMPTY = hoodieError({
   message: 'username must be set',
   status: 400
 })
+
+module.exports.TOKEN_TYPE_INVALID = hoodieError({
+  name: 'Bad Request',
+  message: 'Type must be a string of lowercase characters, numbers, -, or _, and must begin with a character.',
```

Comment:
Can we change the error message to: `Token type must consist of lowercase letters (a-z), digits (0-9), _ and - only. Must begin with a letter`. No `.` at the end